src/main/java/com/autosale/shop/config/RedisObjectMappingConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package com.autosale.shop.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.serializer.GenericJackson2JsonRedisSerializer;
+
+@Configuration
+public class RedisObjectMappingConfiguration {
+
+    @Bean
+    GenericJackson2JsonRedisSerializer userSessionRedisSerializer() {
```

Comment:
Do we really need this? Probably we can use objectMapper?

Reply:
Changed to ObjectMapper but to ignore JsonProcessingException (which is not really so important https://github.com/FasterXML/jackson-databind/issues/723) I marked methods with SneakyThrows annotations

Reply:
But it still throws :) So, I recommend handling an exception